### DISCUSSION TITLE
Sound emitted only on Windows

### DISCUSSION THREAD

**Discussion by Fizayy** [lavalink-devs/Lavalink#1064] [Q&A] [answered]
Hello,

I'm testing my bot on Windows 11 with local Lavalink, the sound is emitted correctly, I can hear the song.
But on my Debian 12 VPS, song is starting but the sound is not emitted, I can't hear something.
It's the same configuration, but I noticed these logs that they are not appearing on my VPS but on Windows:

**2024-07-04T19:49:16.993+02:00  INFO 4208 --- [back-1-thread-1] c.s.l.c.natives.NativeLibraryLoader      : Native library libmpg123-0: loading with filter com.sedmelluq.discord.lavaplayer.natives.ConnectorNativeLibLoader$$Lambda/0x00000265b8815a28@1b756c14
2024-07-04T19:49:17.025+02:00  INFO 4208 --- [back-1-thread-1] c.s.l.c.natives.NativeLibraryLoader      : Native library libmpg123-0: successfully loaded.
2024-07-04T19:49:17.026+02:00  INFO 4208 --- [back-1-thread-1] c.s.l.c.natives.NativeLibraryLoader      : Native library connector: loading with filter null
2024-07-04T19:49:17.189+02:00  INFO 4208 --- [back-1-thread-1] c.s.l.c.natives.NativeLibraryLoader      : Native library connector: successfully loaded.**

Thanks for help.

**Comment by Fizayy** [comment]
Someone know why ?

**Comment by aikaterna** [comment]
Can you post your whole spring.log from startup till you try playing a track on the Debian system? Can't tell much without logs.

**Comment by Fizayy** [comment]
[spring.log](https://github.com/user-attachments/files/16202745/spring.log)
This is my spring.log

**Comment by Fizayy** [comment, marked as answer]
I think it's contabo problem, not detecting native library, i just tested on friend OVH VPS, it's working well...

**Reply by Fizayy** [reply]
I switched on an another VPS and it's working, so on Contabo it's not working :)

**Reply by roselanesan** [reply]
Hi, i wanna ask something about LL with contabo, are you using yt plugins? and if it yes, does it work when you enabling the ipv6? you could dm's me on discord @1t9
